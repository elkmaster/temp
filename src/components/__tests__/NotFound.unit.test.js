import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../NotFound';

describe('NotFoundPage component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot()
  });
});

