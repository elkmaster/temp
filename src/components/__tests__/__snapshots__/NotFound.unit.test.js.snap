// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component should render correctly 1`] = `
<p>
  Not found 
</p>
`;
